admin/views/school/select-pupils-not-taking-check.ejs

```diff
@@ -35,37 +35,95 @@
-                        <div>Pupil has not taken the check due to absence</div>
-                    </li>
-                    <li>
-                        <div><strong>Left school</strong></div>
-                        <div>Pupil has left the school</div>
-                    </li>
-                    <li>
-                        <div><strong>Incorrect registration</strong></div>
-                        <div>Pupil has been incorrectly added to the Pupil Register. They may have been added by the school, or incorrectly included in the pre-populated list</div>
-                    </li>
-                    <li>
-                        <div><strong>Withdrawn</strong></div>
-                        <div>Pupil has been withdrawn from the check.</div>
-                        <div>This includes pupils who:</div>
-                        <ul>
-                            <li>have just arrived and you have not had time to assess their abilities</li>
-                            <li>have no understanding of multiplication tables</li>
-                            <li>are unable to access the check</li>
-                        </ul>
-                    </li>
-                </ul>
-            </details>
-        </div>
+            <div class="master-detail text">
+                <details>
+                    <summary><span class="summary"> What do these reasons mean? </span></summary>
+                    <ul class="panel-no-border attendance-reasons">
+                        <li>
+                            <div><strong>Absent</strong></div>
+                            <div>Pupil has not taken the check due to absence</div>
+                        </li>
+                        <li>
+                            <div><strong>Left school</strong></div>
+                            <div>Pupil has left the school</div>
+                        </li>
+                        <li>
+                            <div><strong>Incorrect registration</strong></div>
+                            <div>Pupil has been incorrectly added to the Pupil Register. They may have been added by the school, or incorrectly included in the pre-populated list</div>
+                        </li>
+                        <li>
+                            <div><strong>Withdrawn</strong></div>
+                            <div>Pupil has been withdrawn from the check.</div>
+                            <div>This includes pupils who:</div>
+                            <ul>
+                                <li>have just arrived and you have not had time to assess their abilities</li>
+                                <li>have no understanding of multiplication tables</li>
+                                <li>are unable to access the check</li>
+                            </ul>
+                        </li>
+                    </ul>
+                </details>
+            </div>
+
+            <br class="clearfix"/>
```

Comment:
Please use css to apply spacing where needed.  We shouldn't be using repeated `br`s

Reply:
Done.